Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -489,6 +489,35 @@ def audit_conflicts
     EOS
   end
 
+  def audit_keg_only_style
+    return unless @strict
+    return unless formula.keg_only?
+
+    # This should catch most things, except awkwardness like
+    # LibreSSL or OpenSSL, which we can manually catch.
+    formula_name = formula.name.sub(/^./, &:upcase)
```

Comment:
Thanks for the clarification! I wasn't entirely sure what you were asking but I wasn't sure if I was just being slow 😸. I'll take a peek at doing this a little differently.